src/index.html

```diff
@@ -7,12 +7,66 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="card">
+    <div
+      class="card__container"
+      data-qa="card"
+    >
+      <div class="image"></div>
+
+      <div class="card__name">
+        APPLE A1419 iMac 27" Retina
+        <br />
+        5K Monoblock (MNED2UA/A)
+      </div>
+
+      <div class="card__article">Product code: 195434</div>
+
+      <div class="review">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="review__count">Reviews: 5</div>
+      </div>
+
+      <div class="card__price">
+        <div class="card__price-text">Price:</div>
+        <div class="card__price-cost">$2,199</div>
+      </div>
+
+      <div
+        class="buy"
+        href="#"
+        data-qa="hover"
+      >
+        <button
+          type="submit"
+          class="button"
+        >
+          Buy
+        </button>
+      </div>
```

Comment:
no needs to wrap with `div`
```suggestion
      <button
        type="submit"
        class="card__button"
        data-qa="hover"
      >
        Buy
      </button>
```